doc/md/language-manual.md

```diff
@@ -467,6 +467,7 @@ The syntax of an *expression* is as follows:
   <exp> . <nat>                                  tuple projection
   ? <exp>                                        option injection
   { <exp-field>;* }                              object
+  { <exp> (and <exp>)* (with <exp-field>;+)? }   object combination/extension
```

Comment:
Can we say `<exp_post>` here, i.e. `M.Class()` is allowed, but `a <some operator> b` is not.

Reply:
Right, but the manual isn't really talking about the grammar but the ast. This would be the only place it does that place.

Reply:
That's fine, then.